sepolicy/Android.mk

```diff
@@ -0,0 +1,109 @@
+# Board specific SELinux policy variable definitions
+ifeq ($(call is-vendor-board-platform,QCOM),true)
+BOARD_SEPOLICY_DIRS += \
+       device/t2m/flame/sepolicy/common \
+       device/t2m/flame/sepolicy/test \
+       device/t2m/flame/sepolicy/$(TARGET_BOARD_PLATFORM)
+
+BOARD_SEPOLICY_UNION += \
+       genfs_contexts \
+       file_contexts \
+       service_contexts \
+       property_contexts \
+       te_macros \
+       device.te \
+       vold.te \
+       ueventd.te \
+       file.te \
+       property.te \
+       untrusted_app.te \
+       drmserver.te \
+       adbd.te \
+       app.te \
+       cnd.te \
+       system_server.te \
+       mediaserver.te \
+       msm_irqbalanced.te \
+       qmuxd.te \
+       netmgrd.te \
+       port-bridge.te \
+       atfwd.te \
+       radio.te \
+       smd_test.te \
+       qmi_ping.te \
+       qmi_test_service.te \
+       irsc_util.te \
+       netd.te \
+       rild.te \
+       diag.te \
+       diag_test.te \
+       audiod.te \
+       service.te \
+       system_app.te \
+       thermal-engine.te \
+       vm_bms.te \
+       system_app.te \
```

Comment:
No, I didn't review the files of the first commit, since I just copied the files from git://codeaurora.org/device/qcom/sepolicy, because the plan was to keep all device specific policies in one place.